src/app/(app)/(home)/support/support-client.tsx

```
@@ -30,6 +29,35 @@ import { renderToText } from '@/lib/utils';
+      const h = window.location.hash.replace('#', '');
+      if (h === 'buyers' || h === 'sellers') setTab(h);
+    };
+    window.addEventListener('hashchange', onHash);
+    onHash();
+    return () => window.removeEventListener('hashchange', onHash);
+  }, []);
+
+  // when user switches tabs, update the hash without jumping
+  const handleTabChange = (value: string) => {
+    const next = value === 'sellers' ? 'sellers' : 'buyers';
+    if (typeof window !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)